src/Pos/Launcher/Runner.hs

```diff
@@ -220,6 +223,7 @@ runServer mkTransport mkReceiveDelay mkL (OutSpecs wouts) withNode afterNode oq
             mkListeners mkL' ourVerInfo theirVerInfo
     stdGen <- liftIO newStdGen
     logInfo $ sformat ("Our verInfo: "%build) ourVerInfo
+    notifyReady
```

Comment:
Ah no, that's not what we want... In fact I think #1318 needs to be changed. The delay should take place within the `NodeAction` continuation